security/tutorials/app-layer-policy/enforce-policy-istio.md

```diff
@@ -237,10 +292,11 @@ account for the customer web app.
 
 ```yaml
 apiVersion: projectcalico.org/v3
-kind: GlobalNetworkPolicy
+kind: NetworkPolicy
 metadata:
-  name: database
+  name: default.database
```

Comment:
The policy in the yaml used above are GlobalNetworkPolcy, so not sure why we are changing this to NetworkPolicy.

Reply:
fixed.